pkg/sql/logictest/testdata/logic_test/create_table

```diff
@@ -340,13 +341,28 @@ SHOW CREATE TABLE like_hash
 ----
 like_hash  CREATE TABLE public.like_hash (
            a INT8 NULL,
-           crdb_internal_a_shard_4 INT4 NOT VISIBLE NOT NULL AS (mod(fnv32(COALESCE(CAST(a AS STRING), '':::STRING)), 4:::INT8)) STORED,
```

Comment:
so this is because `INCLUDING DEFAULTS` isn't in there, wonder if that's passable.